modules/web/src/app/cluster/details/cluster/rbac/users-or-groups/component.ts

```diff
@@ -0,0 +1,119 @@
+// Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Component, OnDestroy, OnInit, Input, Output, EventEmitter} from '@angular/core';
+import {RBACService} from '@core/services/rbac';
+import {Subject, combineLatest} from 'rxjs';
+import {takeUntil, map} from 'rxjs/operators';
+import {MatTableDataSource} from '@angular/material/table';
+import {Cluster} from '@shared/entity/cluster';
+import {ClusterBinding, Binding, SimpleClusterBinding, Kind} from '@shared/entity/rbac';
+
+enum BindingMode {
+  Cluster = 'Cluster',
+  Namespace = 'Namespace',
+}
+
+enum Column {
+  Scope = 'Scope',
+  User = 'User',
+  Group = 'Group',
+  ClusterRole = 'Cluster Role',
+  Namespace = 'Namespace',
+  Actions = 'Actions',
+}
+
+@Component({
+  selector: 'km-rbac-users-or-groups',
+  templateUrl: './template.html',
+})
+export class RBACUsersOrGroupsComponent implements OnInit, OnDestroy {
+  private _unsubscribe = new Subject<void>();
+  readonly Column = Column;
+  readonly RBACType = Kind;
+  columns = [Column.Scope, Column.User, Column.ClusterRole, Column.Namespace, Column.Actions];
+
+  isLoading = true;
+  dataSource = new MatTableDataSource<SimpleClusterBinding>();
+  clusterBindings: ClusterBinding[] = [];
+  namespaceBindings: Binding[] = [];
```

Comment:
We're not using these anywhere (other than assigning values to them) so can we remove these?